### PR TITLE
Make ZIO microsite look really nice

### PR DESCRIPTION
**Pull request by Vilkina**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ target
 project/.sbt
 # if you are here to add your IDE's files please read this instead:
 # https://stackoverflow.com/questions/7335420/global-git-ignore#22885996
+.DS_Store
```
Comment (regiskuckaertz) [inline .gitignore]: This needs to go in your personal gitignore file, see [here](https://stackoverflow.com/questions/7335420/global-git-ignore#22885996) for instructions.

File: build.sbt [hunk 2]
```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
...
     ),
-    micrositeName := "ZIO",
-    micrositeDescription := "ZIO",
+  /*  micrositeCssDirectory := "/microsite/resources/microsite/css/main.css",*/
+
+    micrositeName := "",
+    micrositeDescription := "Type-safe, compositional concurrency for Scala",
     micrositeAuthor := "ZIO contributors",
-    micrositeOrganizationHomepage := "https://github.com/scalaz/scalaz-zio",
+    micrositeOrganizationHomepage := "http://localhost:4000",
```
Comment (rssh) [inline build.sbt]: Looks, like original url should be there instead localhost

Comment (regiskuckaertz) [inline build.sbt]: That looks wrong.

File: build.sbt [hunk 3]
```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
...
     micrositeGitterChannelUrl := "scalaz/scalaz-zio",
-    micrositeGitHostingUrl := "https://github.com/scalaz/scalaz-zio",
+    micrositeGitHostingUrl := "http://localhost:4000",
```
Comment (rssh) [inline build.sbt]: the same.  (hostingUrl)

Comment (regiskuckaertz) [inline build.sbt]: That looks wrong.

File: .gitignore (hunk 1 above)
Comment (Vilkina) [reply .gitignore]: how to delete from this pull?

File: build.sbt (hunk 2 above)
Comment (Vilkina) [reply build.sbt]: I see

Comment (Vilkina) [reply build.sbt]: tnx)

File: .gitignore (hunk 1 above)
Comment (regiskuckaertz) [reply .gitignore]: Just delete the line and commit 😃

Comment (jdegoes) [reply .gitignore]: Yes, just delete this single line from the `.gitignore` file (you can add it to your personal one).

File: build.sbt [hunk 4]
```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
         |<p>&copy; 2018 <a href="https://github.com/scalaz/scalaz-zio">ZIO Maintainers</a></p>
         |""".stripMargin
     ),
-    micrositeName := "ZIO",
-    micrositeDescription := "ZIO",
+  /*  micrositeCssDirectory := "/microsite/resources/microsite/css/main.css",*/
```
Comment (jdegoes) [inline build.sbt]: This can be deleted.

File: build.sbt [hunk 5]
```diff
@@ -145,22 +146,25 @@ lazy val microsite = project.module
...
     micrositeGitterChannelUrl := "scalaz/scalaz-zio",
     micrositeGitHostingUrl := "https://github.com/scalaz/scalaz-zio",
     micrositeGithubOwner := "scalaz",
     micrositeGithubRepo := "scalaz-zio",
     micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512")),
+    micrositeHighlightTheme := "atom-one-light",
     micrositeDocumentationUrl := s"https://javadoc.io/doc/org.scalaz/scalaz-zio_2.12/${(version in Compile).value}",
     micrositeDocumentationLabelDescription := "Scaladoc",
     micrositeBaseUrl := "/scalaz-zio",
     micrositePalette := Map(
```
Comment (jdegoes) [inline build.sbt]: Are these colors being used, or are they overridden in the CSS file?

Comment (Vilkina) [reply build.sbt]: overriden in css file

Comment (Vilkina) [reply build.sbt]: We don't need this line

**Comment by Vilkina** [comment]
@regiskuckaertz , unused styles removed. 
Checked locally - works fine.  https://github.com/scalaz/scalaz-zio/pull/362/commits/f6607662cea770beca7b479f8a5fbd2d1ec8ff48
Image resized.

Ds_Store line deleted.
micrositeOrganizationHomepage changed.
empty lines from build.sbt deleted.
comments from build.sbt deleted

Why I have this error /home/travis/build/scalaz/scalaz-zio/build.sbt isn't formatted properly! - https://travis-ci.org/scalaz/scalaz-zio/builds/453860306?utm_source=github_status&utm_medium=notification

File: build.sbt (hunk 5 above)
Comment (Vilkina) [reply build.sbt]: @regiskuckaertz , @rssh can you review it again?

**Comment by Vilkina** [comment]
@regiskuckaertz - here you can find a screenshot https://pbs.twimg.com/media/DracK4AWkAEhDjH.jpg:large

**Pull request closed without merging**